pkg/innerring/processors/governance/process_update.go

```diff
@@ -53,12 +53,24 @@ func (gp *Processor) processAlphabetSync() {
 			zap.String("error", err.Error()))
 	}
 
-	// 2. Update NeoFSAlphabet role in side chain.
-	innerRing, err := gp.morphClient.NeoFSAlphabetList()
-	if err != nil {
-		gp.log.Error("can't fetch inner ring list from side chain",
-			zap.String("error", err.Error()))
+	// 2. Update NeoFSAlphabet role/netmap contract in side chain.
+	var innerRing keys.PublicKeys
+
+	if gp.morphClient.ProbeNotary() {
+		innerRing, err = gp.morphClient.NeoFSAlphabetList()
```

Comment:
I suggest to define irFetcher interface in the governance processor and reuse implementation from indexer.

Reply:
https://github.com/nspcc-dev/neofs-node/pull/705/commits/0572d3becaa26f475cff9fb6b17c5de9e9082042